src/pages/DogProfileSettingsPage/DogProfileSettingsPage.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import DogProfileSettingsPage from './DogProfileSettingsPage';
+
+describe('Dog Settings Page', () => {
+  it('should have a submit button', () => {
+    render(
```

Comment:
We discovered that the render() function can be called once per describe block instead of inside each test individually. Nothing wrong with doing it this way, but a potential optimization.